swift-sdk/Internal/IterableEmbeddedManager.swift

```diff
@@ -5,6 +5,29 @@
 import Foundation
 import UIKit
 
+public struct ResolvedMessage {
+    public let title: String?
+    public let description: String?
+    public var image: UIImage?
+    public let buttonText: String?
+    public let buttonTwoText: String?
+    public let message: IterableEmbeddedMessage
+
+    init(title: String?,
+         description: String?,
+         image: UIImage?,
+         buttonText: String?,
+         buttonTwoText: String?,
+         message: IterableEmbeddedMessage) {
+        self.title = title
+        self.description = description
+        self.image = image
+        self.buttonText = buttonText
+        self.buttonTwoText = buttonTwoText
+        self.message = message
+    }
+}
+
```

Comment:
Can move to the bottom as it grabs unnecessary attention when looking at EmbeddedManager class.